### DISCUSSION TITLE
Toast and flash

### DISCUSSION THREAD

**Discussion by josejachuf** [emmett-framework/emmett#355] [Q&A] [answered]
There is some way to integrate TOAST [1] with Emmett flash?

Jose

[1}] https://materializecss.com/toasts.html

**Comment by gi0baro** [comment]
@josejachuf since the display of messages should be done by developers (following https://emmett.sh/docs/2.2.x/response#message-flashing), I don't see any particular issue into producing the relative javascript code, eg:

```html
<script type="text/javascript">
  {{ for flash in current.response.alerts(): }}
  M.toast({html: '{{ =flash }}'});
  {{ pass }}
</script>
```

**Comment by josejachuf** [comment, marked as answer]
Thanks @gi0baro
